webpack_config/webpack.freezer.js

```diff
@@ -0,0 +1,22 @@
+// Compile freezer using the (mostly) same webpack config
+'use strict';
+const baseConfig = require('./webpack.base');
+
+const derivationConfig = Object.assign({}, baseConfig, {
+  // Remove the cruft we don't need
+  plugins: undefined,
+  target: undefined,
+  performance: undefined,
+  module: {
+    // Typescript loader
+    loaders: [baseConfig.module.loaders[0]]
+  },
+
+  // Point at freezer, make sure it's setup to run in node
+  target: 'node',
+  entry: {
+    'freezer': './common/freezer'
+  }
+});
+
+module.exports = derivationConfig;
```

Comment:
Are there features that webpack offers that node doesn't have that are valuable enough for the added build time & complexity of this? I feel like node alone should be good enough.

Reply:
Webpack is used here to compile TS to JS, so it's necessary with the current setup. It would be pretty easy though to move to pure JS if that would be preferable.